### ISSUE TITLE
Support upcoming new REST API

### ISSUE THREAD

**Issue by brennerm** [brennerm/check-mk-web-api#31]
With the release of [checkmk 2.0](https://blog.checkmk.com/announcing-checkmk-2.0-and-its-innovation-release) there will be a new REST API. We should add support for it as soon as it's available.

**Comment by ryanlll3** [comment]
Hello brennerm,

I need to get the list of activated alerting rules, service names which are default or autodicovered.
But I see I couldn't get the list of default thresholds, service names.

Do you think you could help or guide?

Ryan

**Comment by xuanyuanaosheng** [comment]
@brennerm  Any new update about using checkmk 2.X ?

**Comment by brennerm** [comment]
@xuanyuanaosheng Nope, not really working on the library anymore these days, sorry.
